### DISCUSSION TITLE
.Net for iOS => net8.0-ios compatibility

### DISCUSSION THREAD

**Discussion by jbouche-work** [AdamEssenmacher/GoogleApisForiOSComponents#28] [Q&A] [answered]
Hi, does the nuget is compatible with net8.0-ios ?

We have post an issue there :
https://github.com/AdamEssenmacher/GoogleApisForiOSComponents/issues/27

And we still have that problem on several nugets of "adame.firebase.ios" :

`_work/_tool/dotnet/packs/Microsoft.iOS.Sdk/17.2.8053/tools/msbuild/iOS/Xamarin.Shared.targets(146,3): error : No matching framework found inside '/Users/user/.nuget/packages/adame.firebase.ios.core/10.24.0/lib/net6.0-ios16.1/Firebase.Core.resources/leveldb.xcframework'. SupportedPlatform: '/Users/user/.nuget/packages/adame.firebase.ios.core/10.24.0/lib/net6.0-ios16.1/Firebase.Core.resources/leveldb.xcframework', SupportedPlatformVariant: 'ios', SupportedArchitectures: ''. [/Users/user/dMAiOSCINantes1/_work/1/s/Dimo.Gmao.Mobile/Dimo.Gmao.Mobile.iOS/Dimo.Gmao.Mobile.iOS.csproj]
`

We are running that in an Azure Pipeline and we get .Net8 et .Net6 framework but we have still the "No matching framework found inside" serveral xcframework of the packages/adame.firebase.ios. ... framework.

**Comment by bengavin** [comment]
I'm in the process of adding .net8 targets to the packages here.  We've been able to use the .NET 6 in our MAUI projects thus far without issue, but we're not trying to leverage any of the Firebase related 'stuff', so that may be why ours are working properly.

**Reply by bengavin** [reply]
Hrm... the Xamarin.Legacy.Sdk stuff doesn't seem to want to play well with .net8.0-ios, haven't figured out exactly why as of yet, will continue to poke at it when I have time.

**Reply by jbouche-work** [reply]
Hi, we may have a solution, we will keep you informed.

**Comment by jbouche-work** [comment, marked as answer]
Hi,

We have fixed it. 
**It was not a problem of the GoogleApisForiOSComponents nugets.**
**And yes the nugets are compatible with .Net8 for iOS.**

We was building our project with ARMv7 witch is obsolete with .Net 8.
Now we build only with : 
`<MtouchArch>ARM64</MtouchArch>`

We had also to disable AOT for iOS in release : 
`<UseInterpreter>true</UseInterpreter>`

Jérôme.
